ouroboros-consensus-cardano-test/src/Test/Consensus/Cardano/Generators.hs

```diff
@@ -198,17 +198,6 @@ instance c ~ MockCryptoCompatByron
                                 (CardanoBlock c)) where
   arbitrary = arbitraryNodeToNode BlockByron BlockShelley BlockAllegra BlockMary
 
-instance c ~ MockCryptoCompatByron
-      => Arbitrary (WithVersion (HardForkNodeToNodeVersion (CardanoEras c))
-                                (Coherent (CardanoBlock c))) where
-  arbitrary =
-        fmap (fmap Coherent)
-      $ arbitraryNodeToNode
-          (BlockByron   . getCoherent)
-          (BlockShelley . getCoherent)
-          (BlockAllegra . getCoherent)
-          (BlockMary    . getCoherent)
-
```

Comment:
I seem to have gotten carried away on the previous PR; this instance is unnecessary.